css/style.css

```diff
@@ -13,11 +13,31 @@ body {
 }
 
 .logo {
-  display: inline-block;
-  padding: 40px 0;
-  font-family: 'PT Serif', serif;
-  font-weight: bold;
-  font-size: 24px;
+  padding: 40px 0 40px 44px;
+  position: relative;
+  font: 24px/31px 'PT Serif', serif;
+}
+
+.logo__circle {
+  position: absolute;
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+}
+
+.logo__circle_color_dark-blue {
+  background: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/html_abrmhah/pullRequest?prid=6295173)